hack/ginkgo-e2e.sh

```diff
@@ -169,15 +169,43 @@ elif [[ "${E2E_TEST_DEBUG_TOOL}" == "gdb" ]]; then
   program=("gdb")
 fi
 
+if [[ -n "${E2E_REPORT_DIR:-}" ]]; then
+  GINKGO_JSON_REPORT_FILE+="${E2E_REPORT_DIR}/ginkgo_report.json"
+fi
```

Comment:
I agree with @aojea: let's not dump a JSON report unless we have something that needs it.

We run a lot of jobs and costs for storing this additional file can become substantial, in particular when it doesn't get pruned as we do for the JUnit file.

Reply:
removed